changelogs/fragments/9010-edit-gitlab-label-color.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - gitlab_label - add feature to update label's color (https://github.com/ansible-collections/community.general/pull/9010).
```

Comment:
So is this a feature (as said here) or a bugfix (as said in the PR description)? I would say this is a bugfix, since changing the color was likely intended when adding support for color, but just not implemented.

Reply:
Agree with bugfix .